test/routes/taskListTest.js

```diff
@@ -177,6 +177,35 @@ describe('GET /taskList/:prisonNumber', () => {
             });
         });
 
+        context('when there is less 4 weeks for the offenders CRD but the DM approves to continue assessment', () => {
+            it('should display opt out form link', () => {
+                licenceServiceStub.getLicence.resolves({
+                    stage: 'ELIGIBILITY',
+                    licence: {
+                        eligibility: {
+                            excluded: {
+                                decision: 'No'
+                            },
+                            suitability: {
+                                decision: 'No'
+                            },
+                            crdTime: {
+                                decision: 'Yes',
+                                dmApproval: 'Yes'
+                            }
+                        }
+                    }
+                });
+
+                return request(app)
+                    .get('/1233456')
+                    .expect(200)
+                    .expect(res => {
+                        expect(res.text).to.include('/hdc/proposedAddress/optOut/');
+                    });
+            });
+        });
+
         context('when offender is ineligible', () => {
```

Comment:
We will now need tests for when the send button is visible on the tasklist

Reply:
We have 3 possible times that it will be visible:
stage = ELIGIBILITY -> caToRo is allowed transition (normal route)
stage =ELIGIBILITY -> caToDm is allowed transition (new route)
stage =PROCESSING_CA -> caToDm  is allowed transition